src/v2/Apps/Example/ExampleApp.tsx

```diff
@@ -0,0 +1,91 @@
+import React from "react"
+import { AppContainer } from "v2/Apps/Components/AppContainer"
+import { createFragmentContainer, graphql } from "react-relay"
+import { ExampleApp_system } from "v2/__generated__/ExampleApp_system.graphql"
+import {
+  Box,
+  Flex,
+  Join,
+  Separator,
+  Spacer,
+  Text,
+  ThemeProviderV3,
+} from "@artsy/palette"
+import {
+  RouterLink as BaseRouteLink,
+  RouterLinkProps,
+} from "v2/Artsy/Router/RouterLink"
+import { ExampleAppMeta } from "./Components/ExampleAppMeta"
+import { HorizontalPadding } from "../Components/HorizontalPadding"
+import styled from "styled-components"
+
+export interface ExampleAppProps {
+  system: ExampleApp_system
+}
+
+const ExampleApp: React.FC<ExampleAppProps> = ({ system, children }) => {
+  const { month, day, year } = system.time
+
+  return (
+    <ThemeProviderV3>
```

Comment:
The footer is still stuck in V2 this way (this is the reason why it's configured in the route)

Reply:
Ah, yeah I saw that we need to add that key to each route so opted into this, forgetting. Will update so that we can check the top level route tomorrow.